src/com/projectkorra/projectkorra/waterbending/WaterManipulation.java

```diff
@@ -167,7 +167,6 @@ public void moveWater() {
 
 					if (isPlant(this.sourceBlock) || isSnow(this.sourceBlock)) {
 						new PlantRegrowth(this.player, this.sourceBlock);
-						this.sourceBlock.setType(Material.AIR);

```

Comment:
I believe this is a leftover change from your previous, unincluded PlantRegrowth rewrite. It stops plant sources from being set to air once utilized.

Reply:
Yup